cogs/word_filter.py

```diff
@@ -45,6 +46,8 @@ def __init__(self, bot):
         self.bot = bot
         self.lock = Lock()
         self.lockSettings = Lock()
+        self.commandBlacklist = dataIO.load_json("data/word_filter/"
+                                                 "command_blacklist.json")

```

Comment:
For a future refactoring, it would probably be better to take these path strings and make them into constants.